packages/semantic-ui-theme/semantic/src/themes/sharegate/modules/dropdown.overrides

```diff
@@ -1,397 +1,553 @@
+.ui.dropdown.error.selection .menu .selected.item {
+    background-color: @alert50;
+    position: relative;
 }
 
-/* STATE | Checked */
-.ui.dropdown[aria-checked="true"],
-.ui.dropdown .menu [aria-checked="true"].item {
+/* STATE | Selected | Checked */
+.ui.dropdown.selection[aria-checked="true"],
+.ui.dropdown.selection .menu [aria-checked="true"].item {
     background: @cloud50;
     font-weight: 400;
 }
 
-.ui.dropdown[aria-checked="true"]::before,
-.ui.dropdown .menu [aria-checked="true"].item::before {
+.ui.dropdown.selection[aria-checked="true"]::before,
+.ui.dropdown.selection .menu [aria-checked="true"].item::before,
+.ui.dropdown.inline[aria-checked="true"]::before,
+.ui.dropdown.inline .menu [aria-checked="true"].item::before {
     position: absolute;
     content: "";
     display: block;
-    width: .25rem;
+    width: 0.25rem;
     height: 100%;
-    background-color: @primaryColor;
+    background-color: @primary500;
     top: 0;
     left: 0;
 }
 
-/* Skin | Selected */
-.ui.dropdown.selected,
-.ui.dropdown .menu .selected.item {
-    font-weight: 400;
+/* STATE | Selected | Checked | SIZE | Small */
+.ui.dropdown.small.selection[aria-checked="true"]::before,
+.ui.dropdown.small.selection .menu [aria-checked="true"].item::before,
+.ui.dropdown.small.inline[aria-checked="true"]::before,
+.ui.dropdown.small.inline .menu [aria-checked="true"].item::before {
+    width: 0.125rem;
 }
 
-.ui.dropdown > .text {
-    line-height: 1.33333333;
-    font-size: 1.125rem;
+/* STATE | Error | Selected | Checked */
+.ui.dropdown.error.selection[aria-checked="true"],
+.ui.dropdown.error.selection .menu [aria-checked="true"].item {
+    background: @alert50;
 }
 
-.ui.dropdown.small > .text {
-    line-height: 2rem;
-    font-size: 1rem;
+.ui.dropdown.error[aria-checked="true"]::before,
+.ui.dropdown.error .menu [aria-checked="true"].item::before {
+    background-color: @alert500;
 }
 
-.ui.floating.dropdown > .menu {
-    margin-bottom: @floatingMenuDistance !important;
+/*******************************
+    Menu
+*******************************/
+/* Layout | Position */
+.ui.dropdown .visible.menu.transition {
+    margin-top: 0.5rem;
 }
 
-.ui.floating.dropdown .menu > .menu {
-    box-shadow: none!important;
+.ui.dropdown.selection .visible.menu.transition {
+    margin-top: 0;
 }
 
-.ui.search.dropdown > input.search {
-    min-height: 38px;
-    font-size: 1.125rem;
+/* SKIN | select/dropdown */
+.ui.dropdown .scrolling.menu {
+    border-top: 0;
 }
 
-/* LAYOUT | Inline */
-.ui.dropdown.inline > .dropdown.icon {
-    width: 0.75em;
-    margin-left: @scaleAlpha;
-    background-color: @marine200;
-    mask-size: 1.5em 1.5em !important;
+.ui.dropdown .menu {
+    border: none;
+    border-radius: 0.25rem;
+    overflow-x: hidden;
+    overflow-y: auto;
 }
-.ui.dropdown.inline + .dropdown-icon > .icon {
-    width: 1em !important;
-    height: 1em !important;
-    fill: @marine200;
+
+/* LAYOUT | Dropdown */
```

Comment:
How come you have "LAYOUT | Dropdown" twice?

Reply:
derp